routes/signup.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(transaction, callback) {
     form: {
       format: 'html',
       lang: transaction.locale,
-      newsletters: 'mozilla-foundation',
+      newsletters: transaction.newsletters || 'mozilla-foundation',
```

Comment:
I think this should be transaction.newsletters only, and we can set a default in the route validation